seq2science/rules/bam_cleaning.smk

```diff
@@ -124,6 +182,23 @@ rule sieve_bam:
         {params.sizesieve_touch}
         """
 
+if not sieve_bam(config):
+    rule cp_unsieved2sieved:
+        """
+        This simply copies a bam file before that is unsieved, to the final bam directory,
+        in case no sieving is necessary. 
```

Comment:
`Copy a bam file if no sieving is necessary. `

this is the worst I could do

Reply:
:see_no_evil: